conf/modules.config

```diff
@@ -0,0 +1,45 @@
+/*
+ * --------------------------------------------------
+ *  Config file for defining DSL2 per module options
+ * --------------------------------------------------
+ *
+ *   Available keys to override module options:
+ *       args           = Additional arguments appended to command in module.
+ *       args2          = Second set of arguments appended to command in module (multi-tool modules).
+ *       publish_dir    = Directory to publish results.
+ *       publish_by_id  = Publish results in separate folders by meta.id value.
+ *       publish_files  = Groovy map where key = "file_ext" and value = "directory" to publish results for that file extension
+ *                        The value of "directory" is appended to the standard "publish_dir" path as defined above.
+ *                        If publish_files == null (unspecified)  - All files are published.
+ *                        If publish_files == false               - No files are published.
+ *       suffix         = File name suffix for output files.
+ *
+ */
+
+params {
+    modules {
+        'openms_map_aligner_identification' {
+            args = "-model:type linear -algorithm:max_rt_shift ${params.max_rt_alignment_shift} "
+        }
+
+        'openms_comet_adapter' {
+            args = "-precursor_mass_tolerance ${params.precursor_mass_tolerance} -fragment_bin_tolerance ${params.fragment_mass_tolerance} -fragment_bin_offset ${params.fragment_bin_offset} -num_hits ${params.num_hits} -digest_mass_range ${params.digest_mass_range} -max_variable_mods_in_peptide ${params.number_mods} -allowed_missed_cleavages 0 -precursor_charge ${params.prec_charge} -activation_method ${params.activation_method} -variable_modifications ${params.variable_mods.tokenize(',').collect { "'${it}'" }.join(" ") } -fixed_modifications ${params.fixed_mods.tokenize(',').collect { "'${it}'"}.join(" ")} -enzyme '${params.enzyme}' -spectrum_batch_size ${params.spectrum_batch_size} "
+        }
+
+        'generate_proteins_from_vcf' {
+            args= "-t ${params.variant_annotation_style} -r ${params.variant_reference}"
+        }
+
+        'percolator_adapter' {
+            args = "-seed 4711 -trainFDR 0.05 -testFDR 0.05 -enzyme no_enzyme -subset-max-train ${params.subset_max_train} -doc ${params.description_correct_features} "
+        }
+
+        'id_filter' {
```

Comment:
what would actually be great, would be to have a separate fdr_threshold parameter for the process `filter_fdr_for_idalignment`.

this one should be higher (lets say default=0.05) than the final `id_filter` (default=0.01)

Reply:
Fixed, the solution is that the check will be performed on the params.fdr_threshold if this is 0.01, then the value for the filter_fdr_for_idalignment will be changed to 0.05. A small note though, even with a value of 0.05, it could be possible that the procedure is unable to continue (a value of .1 works well though).

Reply:
ok sounds good